### PR TITLE
MicroBit Accelerometer

### PR DESCRIPTION
**Pull request by epeach**
# Description
Implements the accelerometer for the MicroBit

accelerometer

## Links

- [spec]()
- [jira]()

## Testing story
Testing is coming after basic demo.

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

cc: @joshlory

### CODE REVIEW

**Comment by joshlory** [comment]
I'd encourage whoever reviews this to pull it locally and try it with a real device, in lieu of automated testing. Lmk if you need a micro:bit board!

File: apps/src/lib/kits/maker/Accelerometer.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+      case 'pitch':
+        return this.pitch;
+      case 'roll':
+        return this.roll;
+      case 'inclination':
+        return this.inclination;
+    }
+  }
+
+  getAcceleration(accelerationDirection) {
```
Comment (ajpal) [inline apps/src/lib/kits/maker/Accelerometer.js:104]: Should this return a number, not a string?

**Comment by epeach** [comment]
@ajpal PTAL! Thanks!

**Review by jmkulwik — APPROVED** [review]
LGTM!

**Pull request merged**